engine-nextgen/src/main/kotlin/graphql/nadel/validation/util/NadelBuiltInTypes.kt

```diff
@@ -33,18 +33,18 @@ object NadelBuiltInTypes {
         .toSet()
 
     val builtInDirectiveSyntaxTypeNames = sequenceOf<AnyNamedNode>(
```

Comment:
I get it that idiomatic Kotlin says mixed case constants but this change would have been a lot smaller by leaving it as is.

Sometimes legacy is legacy - and has diminishing returns when trying to find perfection

Reply:
Yeah, could've left it alone, but I also do have a gripe with SCREAMING_SNAKE_CASE. I don't like it for readability reasons…